scripts/_lines_of_code.py

```diff
@@ -55,58 +55,46 @@ class ContributedRepository:
-            raise NotContributedError('User have not contributed to this repository.')
-        
+        if (contributed_to.get_commits().totalCount >= 100):
+            execute_bind('<inform>', 'This repository has a lot of commits! This can take some time...')
+
         for commit in contributed_to.get_commits():
             if commit.author.id == user.id:
                 for file in commit.files:
-                    if file.filename in excepted_files:
-                        self.excepted_lines_of_addition_in_contribution += file.additions
-                        self.excepted_lines_of_deletion_in_contribution += file.deletions
+                    if file.filename in excepted_files or file.filename.startswith('node_modules/'):
+                        pass
+                    else:
+                        self.total_lines_of_addition_in_contribution += file.additions
+                        self.total_lines_of_addition_in_contribution += file.deletions
 
-        self.net_addition_in_contribution = self.total_lines_of_addition_in_contribution - self.excepted_lines_of_addition_in_contribution
-        self.net_deletion_in_contribution = self.total_lines_of_deletion_in_contribution - self.excepted_lines_of_deletion_in_contribution
+        if not any([self.total_lines_of_addition_in_contribution, self.total_lines_of_deletion_in_contribution]):
+            raise NotContributedError('User have not contributed to this repository.')
 
 
 class OwnedRepository:
     def __init__(self, owned_repository: Repository, user):
         self.total_lines_of_addition = 0
         self.total_lines_of_deletion = 0
-        self.excepted_lines_of_addition = 0
-        self.excepted_lines_of_deletion = 0
         self.name = owned_repository.name
 
-        for contribution in owned_repository.get_stats_contributors():
-            if contribution.author.id == user.id:
-                for week in contribution.weeks:
-                    self.total_lines_of_addition += week.a
-                    self.total_lines_of_deletion += week.d
-
         if (owned_repository.get_commits().totalCount >= 100):
             execute_bind('<inform>', 'This repository has a lot of commits! This can take some time...')
 
-        # Get all of the additions / deletions of the excepted files
         for commit in owned_repository.get_commits():
             if commit.author.id == user.id:
                 for file in commit.files:
-                    if file.filename in excepted_files:
-                        self.excepted_lines_of_addition += file.additions
-                        self.excepted_lines_of_deletion += file.deletions
+                    if file.filename in excepted_files or file.filename.startswith('node_modules/'):
+                        pass
+                    else:
+                        self.total_lines_of_addition += file.additions
+                        self.total_lines_of_deletion += file.deletions
```

Comment:
don't use the syntax:
```
if some_statement:
    pass
else:
    do_something()
```
Use this instead:
```
if not some_statement:
    do_something()
```
```suggestion
                    if not file.filename in excepted_files or file.filename.startswith('node_modules/'):
                        self.total_lines_of_addition += file.additions
                        self.total_lines_of_deletion += file.deletions
```